editor/import/resource_importer_scene.cpp

```diff
@@ -1776,7 +1776,7 @@ void ResourceImporterScene::_optimize_track_usage(AnimationPlayer *p_player, Ani
 						if (bone_idx == -1) {
 							continue;
 						}
-						skel->get_bone_pose(bone_idx);
+						_ALLOW_DISCARD_ skel->get_bone_pose(bone_idx);
```

Comment:
I'd still add a comment that this is used to force updating the bone pose cache.